packages/vscode/src/extension.ts

```diff
@@ -84,6 +85,27 @@ export async function activate(context: ExtensionContext): Promise<void> {
     },
   }
 
+  // enable fileWatcher to watch .prisma folder inside node_modules
+  if (!isDebugOrTestSession()) {
+    const config: WorkspaceConfiguration = workspace.getConfiguration()
+    const filesWatcherConfig = config.get('files.watcherExclude', '')
+    const value = JSON.parse(filesWatcherConfig)
```

Comment:
JSON.parse throws for `''`. Maybe you can make the default `'{}'`

![image](https://user-images.githubusercontent.com/746482/89272770-461f1980-d65c-11ea-892e-3f6ff7dbab0e.png)